src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -179,11 +179,13 @@ class FacetSidebar extends React.Component {
                 return (
                   <label
                     key={j}
+                    id={`${facet.field}-${valueLabel}`}
                     htmlFor={`${facet.field}-${valueLabel}`}
                     className={`nypl-bar_${percentage}`}
                   >
                     <input
                       id={`${facet.field}-${valueLabel}`}
+                      aria-labelledby={`${facet.field} ${valueLabel}`}
```

Comment:
Yeah...

From my brief experience with staff-picks and new-releases (which had a security bug in an older version), I think that upgrading is not very likely. (A focus on shipping, not infrastructure maintaining). So this will affect other applications too, not just Discovery.

This is regarding accessibility, right? That the text within the label should be free of wrapping tags?

I recommend adding a specific class to the <span> you want to use as the numbers field, which forces things right (among other styling choices).